docs/source/model-registry.rst

```diff
@@ -16,31 +16,32 @@ staging to production), and annotations.
 Concepts
 ========
 
-The MLflow Model Registry is a standalone component that introduces several concepts that
-describe the full lifecycle of an MLflow Model.
+The Model Registry introduces a few concepts that describe and facilitate the full lifecycle of an MLflow Model.
 
 Model
-    An MLflow Model logged with one of the model flavor’s ``log_model`` methods.
+    An MLflow Model is created from an experiment or run that is logged with one of the model flavor’s ``mlflow.model_flavor.log_model`` methods. Once logged, this model can then be registered with the Model Registry.
```

Comment:
Maybe `mlflow.<model_flavor>.log_model` to help clarify that it's not a real snippet that can run?

Reply:
Yes, indeed, that bit is true. Will change it.